jinja2/includes/common_macros.html

```diff
@@ -37,7 +37,10 @@
       <nav>
         <a href="{{ user.get_absolute_url() }}">{{ _('Profile') }}</a>
         |
-        <a href="{{ url('account_logout') }}">{{ _('Sign Out') }}</a>
+          <form action="{{ url('account_logout') }}" method="post">
+            {% csrf_token %}
+            <button class="logout" type="submit">{{ _('Sign Out') }}</button>
+          </form>
```

Comment:
Interesting. It appears [the macro greeting is used nowhere](https://github.com/mozilla/kuma/search?utf8=✓&q=greeting), and the whole macro can be removed. 👍